lib/caqti_connection_sig.mli

```diff
@@ -155,8 +157,9 @@ module type S = sig
     ('b -> (unit, 'e) result future) ->
     'a -> (unit, [> Caqti_error.call_or_retrieve] as 'e) result future
   (** Combining {!call} with {!Response.iter_s}, this sends a request to the
-      database and iterates concurrently over the result rows.  Please see the
-      warning in {!fold_s} about resource usage in the callback. *)
+      database and iterates sequentially over the result rows in a non-blocking manner.
+      Please see the warning in {!fold_s} about resource usage in the callback.
+  *)
```

Comment:
And this is good, except please keep the `*)` on the line above for consistency with other doc-comments.